tasks/notify.py

```diff
@@ -38,6 +41,130 @@
 CUMULATIVE_LENGTH = 10
 
 
+class ExecutionsJobInfo:
+    def __init__(self, job_id: int, failing: bool = True, commit: str = None):
+        self.job_id = job_id
+        self.failing = failing
+        self.commit = commit
+
+    def url(self):
+        return f'{BASE_URL}/DataDog/datadog-agent/-/jobs/{self.job_id}'
+
+    def to_json(self):
+        return {"id": self.job_id, "failing": self.failing, "commit": self.commit}
+
+    @staticmethod
+    def ci_visibility_url(name):
+        return f'https://app.datadoghq.com/ci/pipeline-executions?query=ci_level%3Ajob%20%40ci.pipeline.name%3ADataDog%2Fdatadog-agent%20%40git.branch%3Amain%20%40ci.job.name%3A{name}&agg_m=count'
```

Comment:
In this case I'd not use an f-string and define the URL as a constant with a placeholder `{}` inside and use format in the function

Reply:
Does the new code work for you ?